src/Tribe/RSVP.php

```diff
@@ -704,12 +713,20 @@ public function send_tickets_email( $order_id ) {
 			return;
 		}
 
-		$content     = apply_filters( 'tribe_rsvp_email_content', $this->generate_tickets_email_content( $to_send ) );
-		$headers     = apply_filters( 'tribe_rsvp_email_headers', array( 'Content-type: text/html' ) );
-		$attachments = apply_filters( 'tribe_rsvp_email_attachments', array() );
-		$to          = apply_filters( 'tribe_rsvp_email_recipient', $to );
+		$content     = apply_filters( 'tribe_rsvp_email_content', $this->generate_tickets_email_content( $to_send ), $event_id, $order_id );
+
+		/**
+		 * Filters for RSVP tickets email
+		 * @since TBD added new parameters $event_id and $order_id to the filters below
+		 */
+		$headers     = apply_filters( 'tribe_rsvp_email_headers', array( 'Content-type: text/html' ), $event_id, $order_id );
```

Comment:
See [previous note](https://github.com/moderntribe/event-tickets/pull/392#discussion_r126247604) - the docblock reads like it applies to all the filters, but that's not how it works: the doc parser will associate this with the first filter only.